src/tests/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(tests_SRC
     ProducerDeliveryCbTests.cpp
     ConsumerTests.cpp
     MockMessage.h
+    TdcTime_Convert.cpp
```

Comment:
Renaming to `ConvertTDCTests.cpp` would be more consistent with our other test file names.